plugins/kudo.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kudo
+spec:
+  version: "v0.4.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUDO.
+  homepage: https://kudo.dev/
+  description: |
```

Comment:
Minor nit:
You are using quite long lines, which will lead to strange line-wrapping on smaller terminals. Can you re-format to a width of 80 characters?

Reply:
We can consider making this a lint error if necessary since we have a validator tool.

Reply:
Updated :+1: